public/supabase_broker/squashed.sql

```diff
@@ -419,13 +435,21 @@ $$;
 
 ALTER FUNCTION "toit_artemis"."insert_pod"("_pod_id" "uuid", "_pod_description_id" bigint) OWNER TO "postgres";
 
-CREATE OR REPLACE FUNCTION "toit_artemis"."max_event_age"() RETURNS interval
+CREATE OR REPLACE FUNCTION "toit_artemis"."max_events"() RETURNS integer
+    LANGUAGE "sql" IMMUTABLE
+    AS $$
+    SELECT 100;
+$$;
+
+ALTER FUNCTION "toit_artemis"."max_events"() OWNER TO "postgres";
+
+CREATE OR REPLACE FUNCTION "toit_artemis"."min_event_age"() RETURNS interval
     LANGUAGE "sql" IMMUTABLE
     AS $$
-    SELECT INTERVAL '30 days';
+    SELECT INTERVAL '1 day';
```

Comment:
Maybe it's just me, but I think this might need to be a bit higher. 3 days perhaps?

Reply:
done.